designdocs/permissions.md

```diff
@@ -0,0 +1,87 @@
+# proposal for permissions request
+
+
+### Philosophy
+The user-agent will always be closer to the user than a spec. The user agent has greater access to current conditions than we (the spec writers) ever will. As a philosophy we should give the user agent *as much freedom as possible to “do the right thing”* under whatever circumstances are at the time a web application is started.
+
+### scope
+
+Currently we are having difficulties finding a solution that all parties can agree to. This is about the permissions for various properties in WebXR.  This proposal addresses the following concerns
+
+* what permissions are available to be asked for
+* how granular are these permissions
+* are the permissions granted at page load / application start
+* or are the permissions granted at the time of use.
+
+### stating concerns
+
+Concerns about granting at page load time are
+* users may be requested to approve multiple permissions one after another, creating click fatigue.
+* if all apps request permissions at start, the user has no information to judge whether or not they should grant this, because they haven’t entered the experience yet.
+
```

Comment:
Seems like we're missing an implicit option here of granting requests upon initiation of immersive mode, which is where most of the work thus far has actually been focused. Unless that's what was intended by the "page load time" section above, in which case the opening sentence is misleading. Nobody has, to my knowledge, ever suggested a mechanism which allows page load time permission requests from WebXR.

Reply:
Ah, I got the impression that this was referring to the possible navigation stuff that might be added in the future?

Reply:
Perhaps this section just needs an intro paragraph to indicate that the intention isn't to suggest everything being enumerated in it isn't being proposed, but rather is a summary of the concerns that are scattered elsewhere.

Reply:
sorry. in my head 'page load time' and 'session request time' are the same, because I've been working on pages where the only thing you can do is enter immersive mode.  If we change this to 'session request time' would that address your concerns?